mmv1/products/certificatemanager/Certificate.yaml

```diff
@@ -178,9 +185,21 @@ properties:
         name: dnsAuthorizations
         immutable: true
         description: |
-          Authorizations that will be used for performing domain authorization
+          Authorizations that will be used for performing domain authorization. Either issuanceConfig or dnsAuthorizations should be specificed, but not both.
```

Comment:
I could have used the property `exactly_one_of` to enforce that only one of the two properties should be declared. However, It's not explicitly written in the [API ](https://cloud.google.com/certificate-manager/docs/reference/certificate-manager/rest/v1/projects.locations.certificates#managedcertificate). That's why I only wrote it as a comment in the description.